### PR TITLE
Form in req group view

### PR DESCRIPTION
**Pull request by Anand-Issac**
This PR integrates the Request Group Form into the request group view.

- Added a prop onChange to the Request Group Form 
- When request group updated, the change is reflected in the group view

### CODE REVIEW

File: client/src/components/organisms/RequestGroupForm.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ interface DispatchProps {
 type Props = StateProps &
   DispatchProps & {
     handleClose: () => void;
+    onChange: () => void;
```
Comment (meganniu) [inline client/src/components/organisms/RequestGroupForm.tsx]: to be super obvious, i recommend naming the function `onSubmitComplete`

File: client/src/components/organisms/RequestGroupForm.tsx [hunk 2]
```diff
@@ -292,7 +293,12 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
       else {
 
         updateRequestGroup({ variables: { id: props.requestGroupId, name, description, image, requestTypeNames } })
+          .then((res)=>{
+            console.log(res);
+            props.onChange();
+          })
           .catch((err) => { console.log(err) })
+        //props.onChange();
```
Comment (meganniu) [inline client/src/components/organisms/RequestGroupForm.tsx]: stray comment

File: client/src/components/organisms/RequestGroupForm.tsx [hunk 3]
```diff
@@ -292,7 +293,12 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
       else {
 
         updateRequestGroup({ variables: { id: props.requestGroupId, name, description, image, requestTypeNames } })
+          .then((res)=>{
+            console.log(res);
```
Comment (meganniu) [inline client/src/components/organisms/RequestGroupForm.tsx]: stray console.log

File: client/src/components/organisms/AdminRequestGroupBrowser.tsx [hunk 4]
```diff
@@ -64,6 +67,19 @@ const AdminRequestGroupBrowser: FunctionComponent = () => {
...
+    // when user clicks edit button request type
+    const onOpenEditModal = () => {
+        setEditModalShow(true);
+    }
+
+    const handleEditModalClose = () => {
+      setEditModalShow(false)
+    };
+
+    const onChange = () => {
```
Comment (meganniu) [inline client/src/components/organisms/AdminRequestGroupBrowser.tsx]: perhaps change name to `onEditComplete` or something more description

**Review by meganniu — APPROVED** [review]
Great work @Anand-Issac !
Tested it and it works well :))

**Review by chamod-gamage — APPROVED** [review]
lgtm!

File: client/src/components/organisms/RequestGroupForm.tsx (hunk 1 above)
Comment (Anand-Issac) [reply client/src/components/organisms/RequestGroupForm.tsx]: Done

File: client/src/components/organisms/RequestGroupForm.tsx (hunk 2 above)
Comment (Anand-Issac) [reply client/src/components/organisms/RequestGroupForm.tsx]: Done

File: client/src/components/organisms/RequestGroupForm.tsx (hunk 3 above)
Comment (Anand-Issac) [reply client/src/components/organisms/RequestGroupForm.tsx]: Done

File: client/src/components/organisms/AdminRequestGroupBrowser.tsx (hunk 4 above)
Comment (Anand-Issac) [reply client/src/components/organisms/AdminRequestGroupBrowser.tsx]: Done

**Pull request merged**